executor/runtime/types/types.go

```diff
@@ -500,6 +503,17 @@ type Details struct {
 	NetworkConfiguration *NetworkConfigurationDetails
 }
 
+func (d Details) RenderVersionDetails() string {
+	kernelVersionRaw, err := os.ReadFile("/proc/sys/kernel/osrelease")
+	if err != nil {
+		kernelVersionRaw = []byte(fmt.Sprintf("problem reading kernel version: %v", err))
+	}
+
+	kernelVersion := strings.TrimSpace(string(kernelVersionRaw))
```

Comment:
Can you make this a function? Is there really no golang function for this today?

Reply:
I guess there is this library, but it just does the same thing: https://github.com/zcalusic/sysinfo/blob/v0.9.5/kernel.go#L21